docs/src/main/paradox/azure-storage-queue.md

```diff
@@ -0,0 +1,39 @@
+# Azure Storage Queue Connector
+
+The Azure Storage Queue connector provides as Akka Stream Source and Sinks for Azure Storage Queue integration.
```

Comment:
Would be great with a one sentence what-is-Azure-Storage-Queue here in addition to the link where one can read more.